tests/integration/tests/test-create-baseline.js

```diff
@@ -0,0 +1,17 @@
+/* global browser */
+
+"use strict";
+
+
+describe("Firefox Monitor homepage", function() {
+    this.retries(2);
+
+    beforeEach(function() {
+      browser.url("http://localhost:6060/security-tips");
+    });
+
+    it("should look like normal", function() {
+      browser.saveFullPageScreen("SecurityTips_Page", { /* some options*/ });
+      expect(browser.checkFullPageScreen("SecurityTips_Page", {})).to.equal(0);
```

Comment:
Should this be `.to.be.within(0, 5)` (instead of `.to.equal(0)`)?

Reply:
Oh, or should this use `0` since we're trying to verify the screenshot that we just took on L14 above.?

Reply:
Ah this test shouldn't be added to the repo. I'll remove it